### PR TITLE
Add unstable envvar to disable server keepalives

### PR DESCRIPTION
**Pull request by espadolini**
This PR adds an unstable way (setting the `TELEPORT_UNSTABLE_DISABLE_SERVER_KEEPALIVE` envvar to `yes`) to disable the keepalive ticker in the inventory controller; this can be useful in large clusters to reduce the backend load in exchange for slightly less reliable UX (as the lack of keepalives will result in heartbeats potentially going missing sooner if the connection between a peripheral cluster component and the auth service is flaky).

### CODE REVIEW

**Review by Tener — APPROVED** [review]
We could also provide a knob to tune the heartbeat frequency, with a possible value of `never`, although that would be more complex.

**Comment by espadolini** [comment]
> We could also provide a knob to tune the heartbeat frequency

The correct course of action would probably be to use an `interval.VariableDuration` for the keepalive interval kinda like what we do for instance heartbeats, but it was a bit unclear how that should interact with actual heartbeats (periodic ones, and ones "forced" by label changes), so I went with the quick and dirty unstable master switch approach just in case we end up needing it for large clusters before certain changes land (variable keepalives, better behaved dynamodbbk, CRDB).

**Comment by espadolini** [comment]
Also, it wouldn't be hard to parse a duration out of an envvar, it's just that I don't see us ever using that functionality other than to disable the keepalive altogether anyway.

**Review by fspmarshall — APPROVED** [review]
